shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/data/local/datasource/SurveyLocalDataSource.kt

```diff
@@ -0,0 +1,26 @@
+package co.nimblehq.ic.kmm.suv.data.local.datasource
+
+import co.nimblehq.ic.kmm.suv.data.local.model.SurveyRealmObject
+import io.realm.kotlin.Realm
+import io.realm.kotlin.UpdatePolicy
+import io.realm.kotlin.ext.query
+
+interface SurveyLocalDataSource {
+    fun saveSurveys(surveys: List<SurveyRealmObject>)
+    fun getSurveys(): List<SurveyRealmObject>
+}
+
+class SurveyLocalDataSourceImpl(private val realm: Realm) : SurveyLocalDataSource {
+
+    override fun saveSurveys(surveys: List<SurveyRealmObject>) {
+        realm.writeBlocking {
+            surveys.forEach {
+                copyToRealm(it, updatePolicy = UpdatePolicy.ALL)
```

Comment:
`UpdatePolicy.ALL` policy does not fully clear the old data list I think 🤔

Reply:
Because I'm using `PrimaryKey`, so having `UpdatePolicy.ALL` here supports updating the database's data with the latest data from API.

As I mentioned [here](https://github.com/suho/kmm-ic/pull/102#discussion_r1028806537), it's better to keep the database data. What do you think?